docs/FAQ.md

```diff
@@ -1,19 +1,35 @@
-# Limitations and Common Issues 
+# Frequently Asked Questions
 
-## Unity SDK
-### Headless Mode
-If you enable Headless mode, you will not be able to collect visual 
-observations from your agents.
 
-### Rendering Speed and Synchronization
-Currently the speed of the game physics can only be increased to 100x 
-real-time. The Academy also moves in time with FixedUpdate() rather than 
-Update(), so game behavior tied to frame updates may be out of sync. 
+### Scripting Runtime Environment not setup correctly
 
-## Python API
+If you hasn't switched your scripting runtime version from .NET 3.5 to .NET 4.6, you will see such error message:
 
-### Python version
-As of version 0.3, we no longer support Python 2. 
+```
+Assets/ML-Agents/Scripts/ExternalCommunicator.cs(259,24): error CS1061: Type `System.Text.StringBuilder' does not contain a definition for `Clear' and no extension method `Clear' of type `System.Text.StringBuilder' could be found. Are you missing an assembly reference?
+```
```

Comment:
I think this error is too specific, it might change in the future. I would recommend just stating the error will contain `Are you missing an assembly reference?` or have a generic message like : 
```
Type XXX does not contain a definition for XXX and no extension method XXX of type XXX could be found. Are you missing an assembly reference?
```

Reply:
Well, the problem with making it generic is that there could be other reasons this error and those wouldn't be solved by changing the .NET version.

Reply:
I agree with @jo3w4rd on this, maybe we can change this FAQ doc when that situation changes, but right now I've know a few people who struggled with this specific error message.